docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 web:
   build: .
-  command: mix phoenix.server
+  command: service nginx start && mix phoenix.server
```

Comment:
Ah okay.  Shall I remove the `command` directive completely?

AFAIK though we are not using dicker compose at all for prod CI pipeline (just build) so this file is a bit redundant.